Update blob-inventory.md Handling duplicates (#127354)
Azure/azure-content: articles/storage/blobs/blob-inventory.md

Only blobs that are explicitly deleted appear in reports. Therefore, to obtain a complete listing of all soft-deleted blobs (directory and all child blobs), workloads should delete each blob in a directory before deleting the directory itself.

> ### Handling Duplicates in Blob Inventory

Reviewer: ```suggestion
### Handling duplicates in Blob Inventory
```

> Blob Inventory operates on a distributed system, which means that in rare cases, duplicate blob entries may appear in your reports.

Reviewer: ```suggestion
Blob Inventory operates on a distributed system, which means that in rare cases, duplicate blob entries might appear in your reports.
```

> When post processing your inventory report, if your use case requires unique blob entries, to remove duplicates:

Reviewer: ```suggestion
If when post processing your inventory report, your use case requires unique blob entries, you can use the `Name` field to identify and return only the unique blobs. 
```

> -	Use the `Name` field to identify and return only the unique blobs.

Reviewer: ```suggestion
```

> - If your report includes blob versions, use both the `Name` and `Version ID` fields together to identify and return only the unique blobs and versions .

Reviewer: ```suggestion
If your report includes blob versions, use both the `Name` and `Version ID` fields together to identify and return only the unique blobs and versions.
```
